TypeCobol/Compiler/CodeElements/CodeElement.cs

```diff
@@ -29,6 +30,64 @@ protected CodeElement(CodeElementType type)
         /// </summary>
         public CodeElementType Type { get; }
 
+        /// <summary>
+        /// Describe how the CodeElement is debugged
+        /// </summary>
+        public DebugType DebugMode
+        {
+            get
+            {
+                if (_debugMode != DebugType.Unset) return _debugMode;
+                // detect CodeElement with a mix of Debug and "Normal" lines in debugging mode
+                int consumedTokensCount = ConsumedTokens.Count;
+                if (consumedTokensCount > 1)
```

Comment:
`consumedTokensCount > 0` ? Why test 2 tokens at least ?

Reply:
As a `CodeElement` without `ConsumedTokens` is suspicious, an assertion of `consumedTokensCount > 0` has been used.